src/windows-emulator/process_context.hpp

```diff
@@ -79,7 +80,10 @@ struct process_context
     uint64_t current_ip{0};
     uint64_t previous_ip{0};
 
+    uint64_t shared_section_address{0};
+    size_t shared_section_size{0};
     uint64_t dbwin_buffer{0};
+    size_t dbwin_buffer_size{0};
```

Comment:
i'd rather prefer this to be a uint64_t and then static cast it to size_t, where the warnings were, as this causes 32 and 64 bit serialized buffers to stay identical